src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -135,6 +136,9 @@ private async ValueTask InitializeAsync(PackageManagerModel model, INuGetUILogge
             ApplySettings(settings, Settings);
             _initialized = true;
 
+            NuGetExperimentationService = await ServiceLocator.GetComponentModelServiceAsync<INuGetExperimentationService>();
```

Comment:
Why do we need to keep a reference of experimentation service since exp. check happens only once ?

Reply:
Yes, IsRecommenderFlightEnabled depends on it. Could consider refactoring but that would be out of scope of this PR.